src/pages/Facility/settings/chargeItemDefinitions/ChargeItemDefinitionForm.tsx

```diff
@@ -605,10 +605,10 @@ export function ChargeItemDefinitionForm({
           </CardHeader>
           <CardContent className="space-y-6">
             {/* Base Price and MRP */}
-            <div className="p-4 bg-gray-50 rounded-lg border">
+            {/* <div className="p-4 bg-gray-50 rounded-lg border">
```

Comment:
Is there a corresponding closing div? What happens if someone uncomments this? Not an issue though, something to keep in mind for future.